samples/buck/enable_bucks_using_pin/src/main.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2022-2023 Nordic Semiconductor ASA
```

Comment:
```suggestion
 * Copyright (c) 2023 Nordic Semiconductor ASA
```

Reply:
Please run pre-commit for all affected files. A common mistake is the year in the license header.